cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -41,9 +41,19 @@ func (w *wizard) promptOrchestratorConfig() (float64, float64, *big.Int, string)
 	fmt.Printf("Enter fee share percentage (default: 5) - ")
 	feeShare = w.readDefaultFloat(5.0)
 
+	// TODO (nico): Remove pricePerSegment
```

Comment:
Perhaps in the interim before we update the BondingManager contract, to avoid confusion we can:

- Remove all references to `pricePerSegment` in the CLI
- Update the `/activateOrchestrator` endpoint in `server/webserver.go` to *not* check for a `pricePerSegment` in the request and when it makes the `BondingManager.Transcoder()` contract call it hardcodes `pricePerSegment = 0`.

Reply:
So we don't have to worry about breaking changes here because in general people won't be using master with the current protocol implementation, or what is the assumption here?

Reply:
`master` isn't compatible with the protocol live on mainnet right now so its fine to remove references to price per segment except for when the node calls `BondingManager.Transcoder()` - we can change that when we update the contract.

Reply:
Thanks for clarifying !